tests/phpunit/Civi/AllTests.php

```diff
@@ -47,7 +47,7 @@ class Civi_AllTests extends CiviTestSuite {
    */
   private static function getInstance() {
     if (is_null(self::$instance)) {
-      self::$instance = new self;
+      self::$instance = new CiviTestSuite();
```

Comment:
"self" in this case would correspond "Civi\AllTests", not "CiviTestSuite." Testing confirms a regression. (It wasn't caught in PR automated test because that doesn't include Civi\AllTests.) I'm submitting a PR.